### PR TITLE
Feat: update FE to match designs

### PR DESCRIPTION
**Pull request by ghost**
- [ ] Added glowing button for token sale widget + wire-up
- [ ] Added menu item for token sale widget
- [ ] Improved mobile responsiveness for sale widget

### CODE REVIEW

File: components/window.tsx [hunk 1]
```diff
@@ -26,6 +26,8 @@ const TitleBar = ({ hasBorder = true, ...rest }: TitleBarProps) => {
   const { border, id, padding } = useWindowContext();
   const { dispatch } = useWindowManager();
 
+  console.log(id);
```
Comment (arlindlinum) [inline components/window.tsx:29]: can you remove this one please

**Comment by arlindlinum** [comment]
@EsterB12 can you check maybe we don't need to add all of those .png files like the placeholder. We have the background and logo already so maybe we can just position absolute logo to background without adding new png files

File: components/window.tsx (hunk 1 above)
Comment (EsterB12) [reply components/window.tsx]: removed! sorry :D

**Comment by EsterB12** [comment]
> @EsterB12 can you check maybe we don't need to add all of those .png files like the placeholder. We have the background and logo already so maybe we can just position absolute logo to background without adding new png files

I replaced some of the files with svgs where I could! So we cut back on the number of pngs a bit

**Review by arlindlinum — APPROVED** [review]

**Pull request merged**